itsy.css

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0px;
+}
```

Comment:
add spaces after bracket `}`
```suggestion
}

```